src/test/java/com/example/rets_api/repository/PropertyRepositoryJPATest.java

```diff
@@ -73,6 +74,19 @@ public void should_store_a_PropertyAndSchoolAndRoom() {
         assertNotNull(propertySaved.getLotData());
         assertEquals(propertySaved.getLotData(), propertyToSave.getLotData());
 
+        assertNotNull(propertySaved.getViewData());
```

Comment:
You can use the same method that you used in ViewDataConverterTest (checkNullAllFields_ViewDataEntity) instead replicate the code